src/cert/ssl_cert_provider.cpp

```diff
@@ -146,14 +164,14 @@ class X509Cert
         X509_NAME_add_entry_by_txt(name, "O", MBSTRING_ASC, org.data(), org.size(), APPEND_ENTRY, ADD_RDN);
         X509_NAME_add_entry_by_txt(name, "CN", MBSTRING_ASC, cn.data(), cn.size(), APPEND_ENTRY, ADD_RDN);
         X509_set_issuer_name(x509.get(), name);
+        set_san_name(x509.get(), server_name);
 
         if (!X509_set_pubkey(x509.get(), key.get()))
             throw std::runtime_error("Failed to set certificate public key");
 
         if (!X509_sign(x509.get(), key.get(), EVP_sha256()))
             throw std::runtime_error("Failed to sign certificate");
     }
-
```

Comment:
Oh, I missed this.  Please leave in the white space to better differentiate the methods.